src/utils/tag.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+    IconMore16,
+    IconError16,
+    IconClockHistory16,
+    IconCheckmarkCircle16
+} from '@dhis2/ui';
+
+/**
+ * Returns the appropriate Tag props based on the instance status
+ * @param {string} status - The status of the instance
+ * @returns {object} The props to be passed to the Tag component
+ */
+export const getTagProps = (status: string) => {
+    if (!status) return {};
```

Comment:
It seems like formatting hasn't been run. `yarn format` is the key to it. It's beyond the scope of this PR but this should be implemented as a git precommit hook. I created [this](https://dhis2.atlassian.net/browse/DEVOPS-431) Jira task so we can follow up.

Reply:
I tried to implement this recently in this [PR,](https://github.com/dhis2-sre/im-web-client/pull/516) but it was directed to another [ticket](https://dhis2.atlassian.net/browse/DEVOPS-431) created by Andreas previously